google-cloud-logging/.rubocop.yml

```diff
@@ -9,6 +9,7 @@ AllCops:
     - "Rakefile"
     - "support/**/*"
     - "test/**/*"
+    - "lib/google/cloud/logging/rails.rb"
```

Comment:
Seems like the addition to `Google::Cloud::Logging::Railtie.merge_rails_config` method is exceeding the AbcSize limit. I think it's better to just exempt that one method from the AbcSize check, instead of removing the entire file from being linted. Something like this would do:
```ruby
def self.merge_rails_config rails_config # rubocop:disable AbcSize
  ...
end
```